fenix/app/src/main/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckPreferencesMiddleware.kt

```diff
@@ -75,13 +77,17 @@ class ReviewQualityCheckPreferencesMiddleware(
                     reviewQualityCheckPreferences.setEnabled(true)
                     reviewQualityCheckPreferences.updateCFRCondition(System.currentTimeMillis())
                 }
+
+                Shopping.bottomSheetOptInAccepted.record()
+                ShoppingSettings.userHasOnboarded.set(true)
```

Comment:
We should add telemetry events/metrics in a separate middleware focused only on telemetry.